models/intel/human-pose-estimation-0006/description/human-pose-estimation-0006.md

```diff
@@ -14,9 +14,9 @@ The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wr
 
 | Metric                          | Value                                     |
 |---------------------------------|-------------------------------------------|
-| Average Precision (AP)          | 51.2%                                     |
-| GFlops                          | 8.8720                                    |
-| MParams                         | 8.1504                                    |
+| Average Precision (AP)          | 51.1%                                     |
+| GFlops                          | 8.844                                     |
```

Comment:
what the reason to change GFlops and MParams?

Reply:
Just wanted to make them correct :smile: The change is minor, though.